docs/framework/deployment/guide-for-administrators.md

```diff
@@ -225,12 +225,16 @@ This step-by-step article describes how a system administrator can deploy the [!
 ## Troubleshooting  
   
 ### Log file locations  
- The following log files are generated during [!INCLUDE[net_v45](../../../includes/net-v45-md.md)] setup:  
+ The following log files are generated during .NET Framework setup:  
   
- %temp%\Microsoft .NET Framework 4.5*.txt  
- %temp%\Microsoft .NET Framework 4.5*.html  
+ %temp%\Microsoft .NET Framework *version*\*.txt  
+ %temp%\Microsoft .NET Framework *version*\*.html  
   
- You can use the [log collection tool](http://www.microsoft.com/download/details.aspx?id=12493) to collect the [!INCLUDE[net_v45](../../../includes/net-v45-md.md)] log files and to create a compressed cabinet (.cab) file that reduces the size of the files.  
+ where *version* is the version of the .NET Framework that you're installing, such as 4.5 or 4.7.2.  
+ 
+ You can also specify the directory to which log files are written by using the `/log` command-line option in the .NET Framework installation command. For more information, see [.NET Framework deployment guide for developers](deployment-guide-for-developers.md#command-line-options). 
+ 
+ You can use the [log collection tool](http://www.microsoft.com/download/details.aspx?id=12493) to collect the .NET Framework log files and to create a compressed cabinet (.cab) file that reduces the size of the files.  
```

Comment:
nit: the link can be *https*:
https://www.microsoft.com/download/details.aspx?id=12493